services/sidecar/src/simcore_service_sidecar/mpi_lock.py

```diff
@@ -12,111 +12,85 @@
 import asyncio
 import datetime
 import logging
```

Comment:
Can you please update the docstring? I think it is quite out of date

Reply:
done